modules/afsocket/socket-options.c

```diff
@@ -49,6 +49,20 @@ socket_options_setup_socket_method(SocketOptions *self, gint fd, GSockAddr *bind
                           evt_tag_int("so_rcvbuf_set", so_rcvbuf_set));
             }
         }
+      if (self->so_reuseport)
+        {
+#ifdef SO_REUSEPORT
+          if (setsockopt(fd, SOL_SOCKET, SO_REUSEPORT, &self->so_reuseport, sizeof(self->so_reuseport)) < 0)
```

Comment:
Note for the future (since this will be tricky to find otherwise): SO_REUSEPORT does work on WIN32 platforms, but has a very different meaning. Currently it is out of scope.